ds3-sdk/src/main/java/com/spectralogic/ds3client/commands/interfaces/AbstractResponse.java

```diff
@@ -113,6 +113,15 @@ public void checkStatusCode(final int ... expectedStatuses) throws IOException {
         }
     }
 
+    private boolean checkForManagementPortException() {
+        if (this.getStatusCode() == FailedRequestUsingMgmtPortException.MGMT_PORT_STATUS_CODE) {
```

Comment:
That's how I did it originally; then IntelliJ gave me a light bulb and suggested this. Last time I listen to that lying little... I'll go back.